src/cart/crt_iv.c

```diff
@@ -2186,12 +2358,16 @@ crt_ivsync_rpc_issue(struct crt_ivns_internal *ivns_internal, uint32_t class_id,
 	D_ASSERT(input != NULL);
 
 	D_ALLOC_PTR(iv_sync_cb);
-	if (iv_sync_cb == NULL)
+	if (iv_sync_cb == NULL) {
+		D_INFO("NO callback supplied\n");
```

Comment:
This should be error if anything; message is misleading as well

Reply:
Removed D_INFO, message will be handled in exit code.